packages/server/src/adapters/ws.ts

```diff
@@ -74,11 +74,7 @@ function parseMessage(
   const { input: rawInput, path } = params;
   assertIsString(path);
   const input = transformer.input.deserialize(rawInput);
-  return {
-    id,
-    method,
-    params: { input, path },
-  };
+  return { jsonrpc, id, method, params: { input, path } };
```

Comment:
We need a test that actually asserts this otherwise I will break it eventually, guaranteed

Here's a websocket raw client test we can use as a base 

https://github.com/trpc/trpc/blob/97039a761b8704b980fe1a62eaadb5af84942754/packages/server/test/websockets.test.ts#L473-L510

Reply:
Will do!

Reply:
#1785